src/utils/work/fetchWork.js

```diff
@@ -182,7 +182,7 @@ const fetchWork = async (
   // If a work response is in s3, it is cacheable
   // (the cacheable or not option is managed in the worker)
   await cache.set(ETag, response);
-  return response;
```

Comment:
I need to do this to return the ETag, so that it can be passed to the worker. Alternative to this would be to generate ETags outside `fetchWork`and pass it into `fetchWork`, but I think the calling (parent) code shouldn't have to know how to generate ETag. Think of this as a "receipt" given to teh calling function, which may or may not use it.